core/storage/base_model/gae_models.py

```diff
@@ -41,7 +41,7 @@
 DELETION_POLICY = utils.create_enum(  # pylint: disable=invalid-name
     'KEEP',
     'DELETE',
-    'ANONYMIZE',
+    'DELETE_AFTER_VERIFICATION',
```

Comment:
No clue what this means. Verification of what?

I suggest you add a single-line comment above each of these explaining clearly what it means.

Reply:
Done. Renamed to `DELETE_AT_END`